### DISCUSSION TITLE
Conductor: Fixed timespan Mode

### DISCUSSION THREAD

**Discussion by Andersenbond** [nasa/openmct#3895] [Community Q&A] [answered]
Hi, 
I recently adapted the spacecraft tutorial and I have a question regarding the Time Conductor. In the [demo ](https://openmct-demo.herokuapp.com/#/browse/example.taxonomy:spacecraft/example.taxonomy:comms.recd?view=plot-single&tc.mode=fixed&tc.timeSystem=utc&tc.startBound=1094893768000&tc.endBound=1372777565000) we can scroll through time by grabbing and moving the time bounds but In my example when I add the Conductor with the mouse I can't drag the time, but only zooming in the current time bound. So my question is how can I config the Conductor to have the drag option instead of the zooming.

The one in the Demo example:
![image](https://user-images.githubusercontent.com/24599058/119800674-49321180-bedd-11eb-9f33-55575b154162.png)

The one on my project:
![image](https://user-images.githubusercontent.com/24599058/119800720-54853d00-bedd-11eb-8e1a-8d5f23b95e3f.png)

Kind regards, 
Ailton Lopes

**Comment by jvigliotta** [comment, marked as answer]
Hi @Andersenbond, so, you should be able to get that same functionality if you hold down the Alt/Option key while hovering over the time conductor, that should let you scroll through instead of zoom. Let me know if that works for you!

**Reply by Andersenbond** [reply]
Awesome, it worked. Thanks for the support @jvigliotta

**Reply by jvigliotta** [reply]
@Andersenbond my pleasure!

**Reply by milesbjackson** [reply]
I too love this feature!
